### PR TITLE
update typings to extend request object with req.files and req.fields

### PR DESCRIPTION
**Pull request by evanshortiss**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Extension of req object](https://github.com/utatti/express-formidable/blob/master/lib/middleware.js#L16)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

File: types/express-formidable/express-formidable-tests.ts [hunk 1]
```diff
@@ -1,9 +1,14 @@
-import express = require("express");
-import expform = require("express-formidable");
+import * as express from "express"
+import * as expform from "express-formidable";
```
Comment (DanielRosenwasser) [inline types/express-formidable/express-formidable-tests.ts]: Hey @evanshortiss, can you revert these, and also remove the lint suppressions from this package's `tslint.json`?

File: types/express-formidable/index.d.ts [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+// Definitions by: Torkild Dyvik Olsen <https://github.com/tdolsen>, Evan Shortiss <https://github.com/evanshortiss>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { RequestHandler } from "express";
+import * as express from "express";
+import { Fields, Files } from "formidable"
+
+// Extend the express request object with attached formidable files and fields
+declare global {
+  namespace Express {
```
Comment (DanielRosenwasser) [inline types/express-formidable/index.d.ts]: Which global Express does this augment? I don't see the namespace within the `express` types.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/express-formidable/index.d.ts (hunk 2 above)
Comment (evanshortiss) [reply types/express-formidable/index.d.ts]: It's this here - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L11

Express core types come from this oddly enough.

File: types/express-formidable/express-formidable-tests.ts (hunk 1 above)
Comment (evanshortiss) [reply types/express-formidable/express-formidable-tests.ts]: Think it's good now 👍 Out of curiosity, I thought `require` was not recommended, is that not the case?

File: types/express-formidable/index.d.ts (hunk 2 above)
Comment (DanielRosenwasser) [reply types/express-formidable/index.d.ts]: Weird. Okay though!

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**